File: python/practica_1.py
#  EJERCICIO 1

"""
1.Escriba un programa que pida dos numeros enteros y que calcule su division 
escribiendo si la division es exacta o no.

"""


# Declaramos dos variables

numero_uno=5
numero_dos=0
if numero_dos == 0:
    print("No se puede dividir por cero")
elif numero_uno % numero_dos == 0 :
    print("La division es exacta")
elif numero_uno % numero_dos == 1:
    print("La division no es exacta")

# otra forma del codigo

dividiendo = 5
divisor = 2

if divisor == 0:
    print("No se puede dividir por cero")
else:
    if dividiendo % divisor == 0:
        print(f"la division es exacta. Cociente : { dividiendo // divisor} ")
    else:
        print(f"La division no es exacta. Cociente :{dividiendo // divisor }"
         f"Resto: {dividiendo % divisor}" 
        
        )

File: Ejercicios_if_php/practica_1.php
<?php

/*
EJERCICIO 1

1.Escriba un programa que pida dos numeros enteros y que calcule su division 
escribiendo si la division es exacta o no.

*/

// definimos dos variables

$numero_1=5;
$numero_2=0;
/*
if ($numero_1%$numero_2 == 0){
    echo "la division es exacta";
}else{
    echo "La division  no es exacta <br/>";
}
*/
// Mejore el programa anterior haciendo que tenga en cuenta que no se puede dividir por cero

if ($numero_2 == 0 ){
    echo "No se puede dividir por cero <br />";
}else if  ($numero_1 % $numero_2 == 0){
    echo "la division es exacta";
}else if ($numero_1 % $numero_2 == 1){
    echo "La division no es exacta";
}


// Otra forma de hacer codigo

$dividendo = 4;
$divisor = 3;
if ($divisor == 0) {
    echo "No se puede dividir por cero";
}else{
   if ( $dividendo % $divisor == 0  ){
           echo "la division es exacta Cociente  : ".$dividendo / $divisor;
   }else{
        echo "la divison no es exacta Cociente ". $dividendo / $divisor;
        echo "Resto". $dividendo % $divisor;
   }
}
?>